fidesctl/src/fidesctl/cli/cli.py

```diff
@@ -140,6 +140,39 @@ def generate_dataset(
     _generate_dataset.generate_dataset(connection_string, output_filename)
 
 
+@click.command()
+@click.pass_context
+@click.argument("source_type", type=click.Choice(["database"]))
+@click.argument("connection_string", type=str)
```

Comment:
I'd like to see something different than a raw connection string as well, at least eventually (not sure around the timing for getting this functionality out), and I really like the idea of decomposing into components 👍🏽  I'm still not confident in having a wider understanding as yet, but if we can imagine having 3 to 5 connection strings to maintain I would prefer to have a config.yml/json that allows me to manage those as strings or env vars as needed and reference the connection by name/key instead

Reply:
in full agreement here that this isn't optimal, but this would be another issue in and of itself and we use other functions that also rely on a user-defined complete connection string

will also definitely need to do some kind of mapping like @SteveDMurphy mentioned (i gave some examples in the issue) but those are outside of the scope of this issue, this is just an MVP and I think its ok to use the full connection string in that context until we expand the feature-set further